cvat-ui/src/components/header/settings-modal/player-settings.tsx

```diff
@@ -220,6 +224,26 @@ export default function PlayerSettingsComponent(props: Props): JSX.Element {
                     </Row>
                 </Col>
             </Row>
+            <Row justify='start'>
+                <Col span={7}>
+                    <Row className='cvat-player-settings-show-original'>
+                        <Col span={24} className='cvat-player-settings-show-original-checkbox'>
+                            <Checkbox
+                                className='cvat-text-color'
+                                checked={showOriginalImage}
+                                onChange={(event: CheckboxChangeEvent): void => {
+                                    onSwitchShowOriginalImage(event.target.checked);
+                                }}
+                            >
+                                Show original image
+                            </Checkbox>
+                        </Col>
+                        <Col span={24}>
+                            <Text type='secondary'> Downloads the original image instead of the compressed version </Text>
```

Comment:
```suggestion
                            <Text type='secondary'> Downloads the original images instead of the compressed if possible</Text>
```

Reply:
I would suggest to add ``if possible`` because in case with videos it is not clear what is ``original``. Data compressed anyway.